### PR TITLE
Update cromwell instance to  n1-standard-2 and update cromwell version to 38

### PR DESCRIPTION
**Pull request by kdbinder**
Adding fixes to address gsutil bugs and memory bugs in wdl_runner

### CODE REVIEW

**Comment by kdbinder** [comment]
This will also require an updated Docker image from the Broad

File: wdl_runner/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # license that can be found in the LICENSE file or at
 # https://developers.google.com/open-source/licenses/bsd
 
-FROM java:openjdk-8-jre
+FROM openjdk:8-jre
```
Comment (aednichols) [inline wdl_runner/Dockerfile]: My understanding is that Oracle will discontinue maintenance updates on their distribution of OpenJDK 8 in April 2019.

A recommended solution I've seen is https://hub.docker.com/r/adoptopenjdk/openjdk8 which is an independent distribution of OpenJDK that will continue receiving updates.

**Comment by cjllanwarne** [comment]
hey @kdbinder just to confirm, do you want a new docker image to update `broad-dsde-outreach/wdl_runner:2018_11_28` (rather than your placeholder `gcr.io/google.com/kbinder-sandbox/wdl-runner-rev`?)

**Comment by kdbinder** [comment]
@cjllanwarne thats correct. We need the docker container in the broad-dsde-outreach repo to be rebuilt with the new Dockerfile.  @ruchim did this last time.

File: wdl_runner/Dockerfile (hunk 1 above)
Comment (kdbinder) [reply wdl_runner/Dockerfile]: I changed to openjdk based on the verbiage from the original [java](https://hub.docker.com/_/java)

> This image is officially deprecated in favor of the openjdk image

**Comment by ruchim** [comment]
@kdbinder gcr.io/broad-dsde-outreach/wdl_runner:2019-04-15, let me know if that works!

**Comment by kdbinder** [comment]
Thanks @ruchim that works.  Updated

**Comment by moschetti** [comment]
@aednichols are you able to review this to merge?

**Pull request merged**